pingouin/pairwise.py

```diff
@@ -664,7 +664,7 @@ def pairwise_tukey(data=None, dv=None, between=None, effsize='hedges'):
     # Critical values and p-values
     # from statsmodels.stats.libqsturng import qsturng
```

Comment:
Even if it's commented out, let's change it.
These would be all the placed Dominic changed, but it's possible new uses have been added. Have you searched the repo for `qsturng` and `psturng`?

Reply:
I changed what dominic originally changed

Reply:
OK, please do a search to double-check that nothing has been added, and please change this commented-out stuff, too.
Then, if their tests pass, feel free to open up a PR to the main repo.

Reply:
and yes, I did check for all uses